public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -125,13 +124,14 @@ export class CloudWatchDatasource
     this.sqlCompletionItemProvider = new SQLCompletionItemProvider(this, this.templateSrv);
     this.metricMathCompletionItemProvider = new MetricMathCompletionItemProvider(this, this.templateSrv);
     this.variables = new CloudWatchVariableSupport(this);
+    this.annotations = CloudWatchAnnotationSupport;
   }
 
   query(options: DataQueryRequest<CloudWatchQuery>): Observable<DataQueryResponse> {
     options = cloneDeep(options);
 
-    let queries = options.targets.filter((item) => item.id !== '' || item.hide !== true);
-    const { logQueries, metricsQueries } = this.getTargetsByQueryMode(queries);
+    let queries = options.targets.filter((item) => item.hide !== true);
```

Comment:
This check looks invalid to me. CW Logs doesn't use `id`, even though it was (incorrectly) part of the `CloudWatchLogsQuery` type.